_variables.scss

```diff
@@ -61,3 +61,49 @@ $lineThick:       2px;
 // line color
 $lineColorLight:  $colorLightSecondary;
 $lineColorDark:   $colorMedium;
+
+
+
+
+/*----- styles for footer redesign ----*/
+$gridGutterWidth: 12px;
+
+$containerMinWidth: 600px;
+$containerMaxWidth: 1344px;
+
+$breakPointsLg: 1024px;
+
+$gridSmColumns: 8;
+$gridLgColumns: 12;
+
+// New color names
+
+$white:       #fff;
+$lightGrey:  #F4F4F4;
+$mediumGrey: #dfdfdf;
+$darkGrey:   #6d6e6e;
+$black:       #000;
+
+$error:       #ba4242;
```

Comment:
same for here. we should use the same error Color. No?. https://github.com/zalora/global-scss/blob/ee50c86b7aa94c14380f428d44e7a139f8fb3127/_variables.scss#L15